jetty-core/jetty-websocket/jetty-websocket-core-server/src/main/java/org/eclipse/jetty/websocket/core/server/internal/ServerUpgradeResponseImpl.java

```diff
@@ -17,14 +17,13 @@
 import java.util.List;
 
 import org.eclipse.jetty.http.HttpFields;
-import org.eclipse.jetty.http.HttpHeader;
 import org.eclipse.jetty.server.Response;
 import org.eclipse.jetty.websocket.core.ExtensionConfig;
 import org.eclipse.jetty.websocket.core.server.ServerUpgradeResponse;
 
 /**
- * Upgrade response used for websocket negotiation.
- * Allows setting of extensions and subprotocol without using headers directly.
+ * Internal implementation of the {@link ServerUpgradeResponse} interface.
```

Comment:
You need to say more than what I can infer from reading `class ServerUpgradeResponseImpl extends Response.Wrapper implements ServerUpgradeResponse`.

Say something about making the wrapped response available during negotiations and prior to the creation of the CompletedUpgradeResponse